apps/web/playwright/event-types.e2e.ts

```diff
@@ -257,19 +257,20 @@ testBothFutureAndLegacyRoutes.describe("Event Types tests", () => {
         expect(await linkElement.getAttribute("href")).toBe(testUrl);
       });
 
-      // TODO: This test is extremely flaky and has been failing a lot, blocking many PRs. Fix this.
-      // eslint-disable-next-line playwright/no-skipped-test
-      test.skip("Can remove location from multiple locations that are saved", async ({ page }) => {
+      test("Can remove location from multiple locations that are saved", async ({ page }) => {
         await gotoFirstEventType(page);
 
         // Add Attendee Phone Number location
         await selectAttendeePhoneNumber(page);
 
         // Add Cal Video location
-        await addAnotherLocation(page, "Cal Video (Global)");
+        await addAnotherLocation(page, "react-select-4-option-0-0");
```

Comment:
This is what ReactSelect gives internally. We are passing datatest-id but it would be the same for both the dropdowns

This is what we have for Cal Video, but it would still match for two locations - location-select-item-integrations:daily

https://github.com/calcom/cal.com/blob/d938e67e650d70c425a4a689a1c2ff9760b9c8bd/apps/web/components/ui/form/LocationSelect.tsx#L39-L43

Reply:
For context, because of transitions, sometimes briefly we have two Dropdowns open both having the text Cal Video (Gloabl)

https://github.com/user-attachments/assets/bd2d553a-c688-456a-8545-98fdc83addff